nemo_text_processing/text_normalization/es/data/abbreviations_nondet.tsv

```diff
@@ -0,0 +1,6 @@
+betyd.	betydning
```

Comment:
it seems this file should be under `sv/data/abbreviations_nondet.tsv`

Reply:
Fixed